app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="elevation">6dp</dimen>
+    <dimen name="translationZ">4dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_15">15dp</dimen>
+    <dimen name="px_1">1px</dimen>
```

Comment:
Android thì sử dụng dp thôi chứ sao lại px

Reply:
em thấy 1dp hơi to ạ